packer/scylla_install_image

```diff
@@ -114,9 +114,24 @@ if __name__ == '__main__':
         run('systemctl enable amazon-ssm-agent', shell=True, check=True)
 
         run('apt-get install -y linux-aws-lts-22.04', shell=True, check=True)
-        kernel_version = subprocess.check_output("apt-cache depends linux-image-aws-lts-22.04 | grep Depends: |awk -F'linux-image-' '{print $2}' | head -n1", shell=True).decode('utf-8').strip()
-        mid = subprocess.check_output("awk '/Advanced options for Ubuntu/{print $(NF-1)}' /boot/grub/grub.cfg", shell=True).decode().strip().split("'")[1]
-        kid = subprocess.check_output(f"awk '/with Linux {kernel_version}/{{print $(NF-1)}}' /boot/grub/grub.cfg | head -n1", shell=True).decode().strip().split("'")[1]
+        kernel_version = subprocess.check_output("apt-cache depends linux-image-aws-lts-22.04 | grep Depends: |awk -F'linux-image-' '{print $2}' | head -n1", shell=True, text=True).strip()
```

Comment:
This can be replace with following python code as well:
```
        depends_list = run('apt-cache depends linux-image-aws-lts-22.04', shell=True, check=True, capture_output=True, encoding='utf-8').stdout.strip()
        match = re.search(r'^\s+Depends: (linux-image-.+-aws)$', depends_list, flags=re.MULTILINE)
        kernel_version = match[1]
```

Reply:
This will return `linux-image-5.15.0-1034-aws` while we need `5.15.0-1034-aws`